spec/lib/api/v3/work_packages/work_package_schema_representer_spec.rb

```diff
@@ -143,6 +143,26 @@
     end
   end
 
+  describe 'overallCosts' do
+    it_behaves_like 'has basic schema properties' do
+      let(:path) { 'overallCosts' }
+      let(:type) { 'String' }
+      let(:name) { I18n.t('activerecord.attributes.work_package.overall_costs') }
+      let(:required) { false }
+      let(:writable) { false }
+    end
+
+    context 'costs disabled' do
+      before do
+        allow(schema.project).to receive(:costs_enabled?).and_return(false)
+      end
+
+      it 'has no schema for budget' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.